src/rgw/rgw_op.h

```diff
@@ -325,6 +326,50 @@ class RGWGetObj_Filter : public RGWGetDataCB
   }
 };
 
+class RGWGetObjTags : public RGWOp {
+ protected:
+  bufferlist tags_bl;
+  bool has_tags;
+ public:
```

Comment:
Just a stylistic nit, but In /general/ I'd like people to start moving toward using default values for class members instead of a single default constructor with an initialization list.